### DISCUSSION TITLE
1.39.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#23220] [General]
Feel free to discuss the [v1.39.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.39.0) release here!

**Comment by corlo** [comment]
Hi, just install this update and i'm pleased to see the add of the Nous B4Z.  I have prepared a new picture for it that i think look more like the other Nous BxZ modules but i don't know how Github pull request work (i've try but too hard for me).
I join the picture here, perhaps someone can help me and put it at the correct place. 
thanks

![10](https://github.com/Koenkk/zigbee2mqtt/assets/1512243/902509cc-2d13-4a6f-81bd-22fd3632dd42)

**Reply by Koenkk** [reply]
Submitting a new picture is easy, this can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `MODEL.png` (update `model` accordingly) 
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Reply by corlo** [reply]
Ok thanks, i've gave it a try

**Comment by terzo33** [comment]
Hi @Koenkk , new Tongue TOWSMR1 doesn't work properly, some sensors are missing and those that work are updated every 5/10 minutes.

the missing sensors are:

On/Off
Temperature threshold
Temperature breaker
Power threshold
Power breaker
Over current threshold
Over current breaker
Over voltage threshold
Over voltage breaker
Under voltage threshold
Under voltage breaker
Power outage memory
Indicator mode
Rate current Value
Earth Leakage Value

Thanks

**Comment by francisp2** [comment]
Is the change of icon for motion a deliberate change ?

![afbeelding](https://github.com/Koenkk/zigbee2mqtt/assets/52328611/ac00f5ad-7feb-4e96-b33b-67c70f8381f4)

Before, they all had the walking man icon.

Switched back to 1.38

![afbeelding](https://github.com/Koenkk/zigbee2mqtt/assets/52328611/7dde70e1-da4e-4191-8fde-e619c6f92151)

They are motion detectors, not occupancy detectors.

![afbeelding](https://github.com/Koenkk/zigbee2mqtt/assets/52328611/1e367a2c-8ff4-4ea6-a927-4028f0d39ac1)

**Reply by N3rdix** [reply]
Justification described in https://github.com/Koenkk/zigbee2mqtt/issues/22896

For me it makes sense, if you want to have different icons you can change them within the entity settings in HA

**Reply by francisp2** [reply]
So no more differentiation between motion sensors and mm-wave based occupancy sensors ? Do not make sense to me, but it seems that is what it is.

**Reply by N3rdix** [reply]
Somewhat true, to distinguish i'd recommend to adjust the device class in HA. Seems the best way.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/70015952/ca13b44e-4fd2-4b43-a4ef-e7dc45d49829)

**Reply by ghoz** [reply]
You can also override the HA discovery on a device or global level 
```yaml
device_options:
  homeassistant:
    occupancy:
      device_class: motion
```

To be honest I'm not sure I agree with the original change either, no way a PIR based motion detection sensor can be considered an occupancy sensor despite the zigbee cluster name ...
not sure how radar / mmwave occupancy are implemented, but maybe we could try to identify at least the PIR based motion sensors as motion rather than occupancy...

**Comment by momodubassin** [comment]
Hello 
With this version 1.39 my devices aren't available
![image](https://github.com/Koenkk/zigbee2mqtt/assets/174686488/2ffbe48a-b9f1-42c2-9fad-73fa5c923e3c)
Do you help me ?

**Reply by N3rdix** [reply]
They were before? Errors in the log?

**Reply by momodubassin** [reply]
Yes they're intalled 2 years ago
I didn't see error in the log but now I restore old version

**Reply by momodubassin** [reply]
I install again and it s ok now ,  maybe version core vs mqtt

**Comment by HollyFredD** [comment]
Anyone having issues between HA & MQTT since the last update ?

**Comment by mpuff** [comment]
i have problems with the latest zigbee2mqtt version, the gui is not working anymore, the devices can be controlled via mqtt.
and also, on the 1.38, the gui is only working for me when I use a browser like edge, with safari no page is loading

**Reply by tsanidisDev** [reply]
I have the same issue, getting the error in supervisor logs:
2024-07-29 13:27:13.356 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host {myhost}:8099 ssl:default [Connect call failed ('myhost', 8099)]

**Comment by momodubassin** [comment]
I have to restore old version core and mqtt 1.37 . After I upgrade core vesion and upgrade in 1.39 mqtt version

**Reply by Matriciel** [reply]
> I have to restore old version core and mqtt 1.37 . After I upgrade core vesion and upgrade in 1.39 mqtt version

Same problem here. I have downgraded ti 1.37 and pairing is OK

**Comment by martyzeq** [comment]
Hi guys, since this version I see in log only `Zigbee2MQTT: undefined` - any ideas please ? 
Only rsyslog is affected
`  log_syslog:`
`    app_name: Zigbee2MQTT`
`    eol: ''`
`    host: 192.168.X.Y`
`    localhost: z2m`
`    path: /dev/log`
`    pid: process.pid`
`    port: 514`
`    protocol: udp4`
`    type: BSD`

**Reply by ghoz** [reply]
Hi, it is fixed in dev : https://github.com/Koenkk/zigbee2mqtt/issues/23241

**Reply by martyzeq** [reply]
> Hi, it is fixed in dev : #23241

great, thx 👍
